languages/purescript/exercises/concept/basics/.docs/after.md

```diff
@@ -0,0 +1,5 @@
+In the introduction, we explained functions and said that every function has one or more arguments. This is not strictly true. Technically all functions have only one argument; this is called [function currying][function-currying]).
```

Comment:
The `after.md` document should contain a summary of everything that this exercise aimed to teach (see [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsaftermd)). As such, all information that is in the `introduction.md` document should also be in this document. You can expand on that information here or write things differently if that makes sense, but all information should be in this document.